gui/wxpython/mapdisp/properties.py

```diff
@@ -327,3 +369,22 @@ def _createDisplayPage(self, parent):
         panel.SetSizer(sizer)
 
         return panel
+
+    def _createStatusBarPage(self, parent):
+        """Create notebook page for statusbar settings"""
+
+        panel = SP.ScrolledPanel(parent=parent, id=wx.ID_ANY)
+        panel.SetupScrolling(scroll_x=False, scroll_y=True)
+        parent.AddPage(page=panel, text=_("Status bar"))
+
+        # General settings
+        sizer = wx.BoxSizer(wx.VERTICAL)
+
+        # Auto-rendering
+        self.projection = ChBProjection(panel, self.mapWindowProperties)
+        sizer.Add(

```

Comment:
Visually it looks like the checkbox on the statusbar page is slightly moved compared to the other page (switch the tabs to see the difference)

Reply:
Corrected.